### DISCUSSION TITLE
Add Quarterly time frame

### DISCUSSION THREAD

**Discussion by cotton1234** [BennyThadikaran/stock-pattern#70] [General]
Hello Benny,

Can you add Quarterly  time frame in the list with daily, weekly, monthly.

Traceback (most recent call last):
  File "/Users/saurabhgarg/shivshakti/stock-pattern/src/init.py", line 497, in <module>
    loader = getattr(loader_module, loader_name)(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/saurabhgarg/shivshakti/stock-pattern/src/loaders/EODFileLoader.py", line 57, in __init__
    raise ValueError(f"Timeframe must be one of {valid_values}")
ValueError: Timeframe must be one of daily, weekly, monthly

Thanks
Saurabh

**Comment by BennyThadikaran** [comment]
Should not be a problem. I'll try it over the weekend and let you know.

**Reply by cotton1234** [reply]
Thanks Benny

**Reply by BennyThadikaran** [reply]
Update to v3.2.0. I have added support for Quarterly scans :fire: 

`py init.py -p all --tf quarterly`

**Reply by cotton1234** [reply]
Thanks Benny!!
